### ISSUE TITLE
[Parameters & Roadmap WG] For a grants round, don't burn unallocated OCEAN, instead recycle

### ISSUE THREAD

**Issue by trentmc** [oceanprotocol/oceandao#49]
**Discord poll:** https://discord.com/channels/612953348487905282/776848812534398986/915360169452535849

**Status quo mechanism**

For any OCEAN not allocated to a given grants round, it's burned.

Details: 
- OceanDAO funds for a given month are from (a) OPF treasury, (b)  1/4 * 51% supply (for the month), and (c) from 95% * Network Revenue (for the month). If (community decides) there aren't enough projects that promise enough growth, then remaining OceanDAO funds are burned. 
- For (a) and (b) this is just burning incoming OCEAN, and for (c) this means buying OCEAN on the open market and burn it. 
- Currently all funds are from (a).  (b) will be coming in next few months. (c) are "there" but won't be significant until significant traction, and to be larger than (a) + (b) it will likely be several years.

**Reasons for Burn**

1. Burning non-allocated funds allocates more efficiently than allocating to projects that the community doesn't want. (Note: recycling is also more efficient than allocating to projects that community doesn't want). 
2. Buy & burn OCEAN (c) drives upwards price pressure. NOTE: burning OCEAN from (a)(b) does _not_ drive upwards price pressure, though the community has the belief that it might

**Reasons against Burn (Problems with Burn)**

1. In next several years, burning doesn't actually help price. Funds from (a)(b) were never in the circulating supply, so burning them doesn't actually improve price (!). Ref Chris Burniske article (Placeholder.vc). Therefore the only burn that's truly legitimate is from (c), but that is years away
2. In next several years, if we recycled not burned, those funds could be put to better use - towards growth
3. Even if burning was from (c), where it does benefit price, burning gives a one-time benefit, whereas funds towards growth gives exponential payoff over time (snowball effect). And this won't have a strong effect for years.
4. Some community members _believe_ burning will drive price in the near term, and they vote "no" a bunch according to those short-term incentives. This places their incentives at odds with community members who want to build & grow value of Ocean / Ocean ecosystem for the medium & long term. (And it actually doesn't make a difference due to funds from (a)(b).)

**Describe the solution you'd like**
Don't burn, rather, put the OCEAN back into the pot that's headed for future community funding

This would solve the problems from burn.

At some point we could reintroduce buy & burn for funds from (c). But that is likely years away.

**Additional context**
Parameters & WG discord channel: https://discord.com/channels/612953348487905282/908016760190537798

**Comment by tmanthey** [comment]
The core mission of the DAO is to fund projects in order to enter a self sustainability ecosystem. So the DAO has a mid-longterm focus. 
First time in R10 we saw votes for token burn, which is a very short term incentive to support the price.
This incentive is currently not aligned with the core mission and focus of the DAO and has a negative impact on projects as it ambiguates no-votes, into "yes for token burn" and "no, for that project". Projects with small yes vote overhang are prone to be voted out in order to increase the burn.
By returning unused funds No-Votes are not weakened but now unambiguously are a vote against a particular project. Therefore I second that proposal to convert token burn into a cap for funding.

**Comment by scottymilat** [comment]
As you've outlined the tokens that get burned mostly come from points 'a' and 'b' which is essentially taking 'potential value that could be added to Ocean' and removing it. 

The one counter argument I could see from the 'burn to increase price' camp is that it 'reduces maximum supply' which could have a perceived price impact. If enough people believe price will increase because there are less tokens (and it gets picked up on social media etc) it may increase demand and therefore price momentarily. 

That said, this would need to be weighed up against the potential benefits of having those tokens come back to later help fuel valuable growth (which IMO should be the focus and more important than momentary spikes in price). 

If this later direction were to be pursued you would want to have some mechanism around making sure this (potentially growing) stack of tokens is put to good use within a timely fashion. Potentially making them available to fund quality projects (as voted by the community as we do now) within a 12 month expiry or something would lead to putting more pressure on attracting quality projects. i.e. if over time the stack of unspent tokens is increasing and we are failing to allocate them to quality projects then the problem is elsewhere. The 'no vote' tokens don't hang around forever waiting for a good project to come along, instead they put more pressure (and incentive) towards attracting quality projects to improve and grow Ocean. 

In summary, amount of Ocean allocated towards monthly DAO projects remains unchanged. All un-allocated tokens go towards an additional balance which grows the amount of funding available to future projects. These tokens expire after X months (e.g. 12 months) and are burned if they don't get allocated.

**Comment by tmanthey** [comment]
"If enough people believe price will increase because there are less tokens (and it gets picked up on social media etc) it may increase demand and therefore price momentarily."
Yes, that's the problem. But as Bruce pointed out correctly that is impossible from a fundamental point of view because the tokens are coming from the treasury which is not part of the total supply. So by burning DAO tokens the supply is never reduced, but less tokens enter the total supply. As total supply is not increased there is no fundamental for an increased value of the token other than joint believes. But the idea of burning that tokens after 12 month I think is interesting.

**Comment by TimDaub** [comment]
Additionally, and that is independent of any parallels drawn to the OCEAN token's price, what needs enumeration in this discussion is that for a particular set of projects, the funding through those tokens, dramatically changed the OCEAN ecosystem dynamic.

All of this is anecdotal, but anyone being a member of the Ocean community for more than 6-9 months is probably aware that the project now has more impact than it had before the oceanDAO funding mechanism.

I doubt that it's easily measurable what I mean as, e.g., it couldn't be represented in website visitors or the numbers on the marketplace. But by having projects directly build on Ocean, many new long-term committed community members have been attracted. The dialogue and communication about "what Ocean is" and what the "new data economy" could become have improved.

The quality of contributions has skyrocketed, and the engagement has risen because many people have been given a chance to think "full-time" about "the new data economy." Without a doubt, that's a better outcome than paying the opportunity costs of either doing nothing or burning those tokens.

I support recycling.

**Comment by MeMent0L** [comment]
## TL;DR: 
Let's avoid burning tokens and invest this money in building (now or later). 
Let's think about better negative signalling and screen-out mechanisms.

## Token Burn
Removing the token burn of unused funds in the OceanDAO is the right thing to do for the ecosystem in my opinion.
 1. It preserves the "war-chest" for the ecosystem for the long term
 2. It avoids expensive learning experiences as we learn our way to setting-up a DAO

## Signalling mechanism / governance
Besides, a lot of assumptions are made on why some people vote "NO" in the OceanDAO. Are we 100% sure that this is tied to expectations that token price would be affected in the short term? 

I've personally never been very fond of the straight "NO" votes. It's been effective in deterring "abusers" or ethically dubious projects. But it doesn't provide much useful signalling besides that. In its current implementation, "NO" votes may not be effective in screening out bad projects, in particular if there are many of them (1-token-1-vote). 

In a world without "NO" : 

### Signalling 
Signalling against projects can be done with more context in Port, this is where I would invite critics to go. 

### Screening out Abusers
We could imagine a different mechanism to avoid abuses of the OceanDAO - such mechanism is going to become increasingly important as "DAO Free riding" is becoming a thing, and we want to protect ourselves from that.  

An example alternative mechanism to "NO" votes to screen out bad projects would be to get a council elected by token holders that has the ability to veto specific proposals if they do not meet ethical standards (we've seen some dubious projects already). 

That council could be composed of, and voted for, only by non-anon community members, in a 1-person-1 vote format for example. That would avoid the issues related to pure coin-weighted voting, which has been failing projects over and over from Lisk to EOS, to pretty much all DAOs that don't have safeguards to this. Quadratic voting helps, but it does not fix.  

That would require some engineering, of course, but making DAOs decision-making more sophisticated is quite likely the way to go anyways, across Web3 projects. Some inspiration can be drawn from Polkadot: https://wiki.polkadot.network/docs/learn-governance#council

All food for thoughts, of course.

**Comment by idiom-bytes** [comment]
#### Background
First, I'll start by sharing some background with previous discussions WRT burning vs. not burning.

> Another thing we should consider for r11: "no"votes don't lead to a burn, rather, if there is insufficient "yes" projects for the allocated funds, then remainder from allocated money is put into OceanDAO treasury for future rounds.

I understand this, but this relatively big. Is there forever no more burning? This impacts the mechanism/tokenomics quite a bit, and might need help with the communication here.

I would still recommend a burn... perhaps a more "controlled burn", where funds do return to treasury, and then based on treasury reserves, performance metrics, and some algo, decides how to optimize for funding & burning.

Right now is a bit chaotic and is a larger number that I expected. I believe it would be dampened after more voter engagement.

Overall, I still recommend some aspect to this mechanic. It does set a floor (we just experienced this live in front of us). Even if it's a smaller amount. I.E. 10% burn 90% return to treasury.

You can see an example of a decentralized proposal towards understanding this between participants by [viewing LobsterDAO's burn proposal for remaining, unclaimed NFTs](https://snapshot.org/#/10b57e6da0.eth/proposal/QmNsfWz7iYBhU28so49oCjVRv2FiDmn1LWykh1cNsB7pMa).

#### I'll address some parts of this discussion

**1. Burning is a mechanism that stands on it's own.** 
It should not be confused for downvoting.
The two are related (downvoting can trigger a burn), but they are mutually exclusive.

**2. There are arguments in Chris Burniske article that I believe are misinterpreted** 
I'd also like to (!). Ref [Chris Burniske article](https://www.placeholder.vc/blog/2020/9/17/stop-burning-tokens-buyback-and-make-instead) (Placeholder.vc) in my next post.

It basically argues that:
- Immediate price action happens on buyback, not burn.
- Removing from circulation happens when participation increases, not due to scarcity or total equity value.
- Burning does not create any new value.
- **However, it does recognize that burning impacts supply. And given decreased emissions/circulation, price can be impacted.**

```Reducing the number of shares may boost the price, but it doesn’t change the overall value of the system.```

#### Burning as a mechanism and game theory
Burning however is a tokenomic/mechanism that leads to a lot of very interesting game theory.
1. Existing participants are incentivized to not let the money burn (i.e. go onboard more participants).
2. Inbound signal for exterior parties to come join the network, and obtain value from it. (i.e. money is burning, come apply)
3. It **does reduces final circulating token supply**. From [Foundation + Network] => [0xd34d] vs. [Foundation + Network] => [GnosisSafe Treasury]. Given constant or increasing demand in this scenario, price volatility = up.

![Screenshot from 2021-11-24 19-01-14.png](https://images.zenhubusercontent.com/601eeb154030993736fc619e/cf383fcd-f671-4849-a99f-96f3da6e3487)

Not burning will also lead to other, different game theory which I believe needs to be recognized.
1. Growing treasury (honeypot) will need to be managed, and governed.
2. New mechanisms need to be developed to increase fund efficiency such that: No Burning > Burning

#### Conclusion
1. I am primarily in support for burning.

I believe this implements a constructive, intrinsic game theory for active participants, to work together to grow the network.

The sibling mechanism to "burning" that is easier to understand is "network bonus"; where all participants on the network benefit if they reach a common goal. Further, the strongest benefits from burning I see have little to do with extrinsic factors. I believe the current discussions and concerns shared from the community, align with the game theory I would expect to see from:
- Burning is leaving participants recognizing the opportunity cost.
- Participants feel the need to deploy these assets & capital.
- Burning is signaling funding / network inefficiencies that need to be addressed.

I believe not burning may delay our progress (reduced pressure), create a false sense of security (increasing treasury), and add more complexity (perhaps conflicts) to governance/finance as a honeypot will need to be managed.

2. The case for not burning.
In the case a treasury managements & related requirements (for it to be executed well), I appreciate Scott's proposal to have this deployed within a certain amount of time.

I am also in support of algorithmic/protocol level definitions for treasury management, with the goal of eliminating governance/treasury overhead.

I worry about the additional complexities, and focus on treasury management, will lead to further focus on extrinsic motivators. Further a culture that is opposite to: intrinsic motivators, focusing on delivering outcomes, and being rewarded via retroactive funding (which [could be accomplished without a treasury](https://port.oceanprotocol.com/t/finalized-votes4lps/817)).

**Comment by trentmc** [comment]
Clarifications:
- This decision is not "forever" because there can be proposals to change it
- Existing mechanisms readily handle the growing treasury
- Regardless of whether burn happens, there's a healthy war chest for the future. However, not burning preserves it better
- While burning impacts _total_ supply, current burning does not impact _circulating_ supply which is what affects price most

Details of clarifications:

> <Roberto> Is there forever no burning? 

No. Because there can always be proposals to change it. (Unless something is put into an immutable smart contract; but let's cross that bridge when we get there). Though as a community we shouldn't be schizophrenic, but rather, we should try to work through this now and then go with it, and only change based on relevant significant new information.

The arguments for "no burn" now are really for funds from OceanDAO sources (a) and (b) in the description. There are good arguments _to_ burn funds from OceanDAO source (c) because it's a _buy_ and burn. But that's a small component now. We can cross that bridge when we get there.

> <Roberto> [Concern:] Growing treasury (honeypot) will need to be managed, and governed.
> <Roberto> New mechanisms need to be developed to increase fund efficiency such that: No Burning > Burning
> <Scott> you would want to have some mechanism around making sure this (potentially growing) stack of tokens is put to good use within a timely fashion. 

I don't believe any new mechanisms are needed here. Here's how I envision it working:
1. In near term, where funds coming from OPF-managed community wallet: non-allocated OCEAN simply stays in the wallet
2. In med term, where funds coming from a vesting smart contract: non-allocated OCEAN is simply deposited back into the wallet, and the wallet follows its usual mechanics, which simply grows the pot of funds available for future rounds (fwiw I'm working on that smart contract now, and it can handle it.)
3. In long term, what if a huge # OCEAN has been continually deposited back into the wallet and it's not being spent down fast enough, not being "put to use in a timely fashion" (ref Scott)? That _would_ be the time to burn. Good news, there's already a mechanism for this too! What it is: >=1 project proposals asks for "grant" funding, where after the project receives the funds, the project _itself_ burns the funds. The community can use existing voting mechanics to decide for itself whether this is a useful deployment of funds, side-by-side with the other proposals.

> <MeMent0L> [Burning] preserves the "war-chest" for the ecosystem for the long term

To clarify, there is a big "war-chest" independent of burn, because 1/4 of the 51% supply is dedicated to OceanDAO. That said, not burning does indeed increase the size of the war chest further. 

> However, [Burniske article] does recognize that burning impacts supply. 

Correct, it does impact _total_ supply. And over the long term (decades) the total OCEAN supply would be lower. However this is decades. In the meantime, it's only _perceived_ price impact, as Scott pointed out.

However if the funds are from _non-circulating_ OCEAN supply - which they currently are - then burning them does _not_ impact _circulating_ OCEAN supply. (And OCEAN price is more a function of circulating supply than total supply.) This is the argument Bruce was making, based on Burniske article.

**Comment by scottymilat** [comment]
The idea of submitting a proposal to burn tokens which the community votes on seems like a pretty elegant solution for now as it would mean we don't have to build something that may need to be re-built later and the voting community gets to decide whether or not to burn those funds.

**Comment by tmanthey** [comment]
> **1. Burning is a mechanism that stands on it's own.** It should not be confused for downvoting. The two are related (downvoting can trigger a burn), but they are mutually exclusive.

_Should not_ in IT is a running gag. It typically indicates with 50%+ probability that what _should not_ happen, does happen in fact. Imo there shouldn't be speculation whether it happens or not. Tokenomics is a completely different concern from DAO grants and therefore should be clearly separated imo.

> ## Burning - Game Theory
> Burning however is a tokenomic/mechanism that leads to a lot of very interesting game theory.
> 
> 1. Existing participants are incentivized to not let the money burn (i.e. go onboard more participants).
> 2. Inbound signal for exterior parties to come join the network, and obtain value from it. (i.e. money is burning, come apply)

These are interesting aspects. Yet I believe the biggest incentive is the grant itself not the burn. Looking back I can say that token burn led to premature applications to the DAO in case there was a low number of participants. The rationale here: "before things get burned I throw in a proposal and let's see if it gets voted or not". Yet, this seems to attract more low quality applications. 

> 3. It **does reduces final circulating token supply**. From [Foundation + Network] => [0xd34d] vs. [Foundation + Network] => [GnosisSafe Treasury]. Given constant or increasing demand in this scenario, price volatility = up.

I think that the grant tokens in relation to the fully diluted supply has no short term impact whatsoever.

So taking these arguments into consideration I stay with my prior opinion that the negative effects of token burn in the DAO outweigh the positive aspects by a margin.

Tokenburn is related to tokenomics. I can imagine there are good reasons to burn tokens. But this should be completely separate from the Grants DAO to not mix different concerns.

**Comment by MeMent0L** [comment]
We should really strive to avoid any confusion between token price and network usage. 

Is token price the optimization metric? 
 - If yes, it would be more effective to burn a big portion of tokens straight away and make holders happy, why bother... 
 - If not, what is likely to drive more network usage?

It's really as simple as that IMO.

@idiom-bytes I would love to explore the game theory models you highlighted above over the weekend; would it be possible to reference them here?

Not burning increases *slightly* the pressure on treasury governance. But as we are at it, I'll give my 2 cents aside:  
*I salute your intent to move away from politics, conflicts and misalocation of funds @idiom-bytes. But do we realistically believe that a voting mechanism can replace functioning democratic systems with powers and counter-powers to reach the best funds-allocation outcome? I wouldn't bet the house on a mechanism that looks like an improved version of plutocracy... 
More importantly, governance is not the particular voting system that will end up being implemented, it's this discussion here, and the many others that shape how the ecosystem evolves. We really ought to recognize that. But that's another issue.*

I personally only know BNB as a successful implementation of a *usage/revenue based burning mechanism*. They did not over-engineered the burn; tx fees from users get burned.

Simple rules are typically the most effective, as they are easier to make right, understand and communicate. In FMCG for example, some experts argue that providing rewards (e.g. 10% discounts on next purchase) based on a complex AI algorithm is less effective than simple rules (e.g. Buy 10 get the 11th for free) because complexity is perceived as arbitrary by most people...

Anyways, if you have other examples it would help with context.

**Comment by idiom-bytes** [comment]
> In near term, where funds coming from OPF-managed community wallet: non-allocated OCEAN simply stays in the wallet
> In med term, where funds coming from a vesting smart contract: non-allocated OCEAN is simply deposited back into the wallet, and the wallet follows its usual mechanics, which simply grows the pot of funds available for future rounds (fwiw I'm working on that smart contract now, and it can handle it.)

I believe you have addressed Idiom-(5) "How to reduce any sort of treasury and governance." which was a key point I wanted to made sure it was sorted and communicated clearly. 

Now: Unallocated funds stays in OPF wallet.
Future: Unallocated vesting funds move to OPF wallet. Unallocated funds stays in OPF wallet.
There is nothing else.

> The community can use existing voting mechanics to decide for itself whether this is a useful deployment of funds, side-by-side with the other proposals.

I believe this could be said for many things.
There are many native/protocol decisions that are not addressed by governance.

I would rather entrust a team of people modelling these parameters, and mechanisms that go beyond burning (i.e. earmarks), than perhaps putting it to a vote. :smile:

Having said this I'll just refer back to my post so I can communicate more effectively. Idiom-(1)(2) 

When I talk about a burn, I'm talking about a controlled amount, to still have an intrinsic mechanism to drive token holders and network participants, towards full capital deployment and network growth. Idiom - (6)

> Is token price the optimization metric?

I believe our North Star is to work together and grow network participation. Idiom-(6)

> If not, what is likely to drive more network usage?

Burning natively forces us to figure out ways to increase network usage.
It is the only way to unlock more value.
It is our Level-1 Moloch.

> They did not over-engineered the burn; tx fees from users get burned.

Ocean is the same. 
There is a % on network revenue that gets burned.

The other burn happens in the grants program when we fail to fund enough projects.
In this case, 100% unused funds are burned.

> I think that the grant tokens in relation to the fully diluted supply has no short term impact whatsoever.

I never brought up fully dilluted supply. 
I am specifically referencing Circulating + Non-circulating tokens entering circulation. - Idiom(5)

**Comment by MeMent0L** [comment]
So, the optimization metric is participation in OceanDAO?

**Comment by idiom-bytes** [comment]
> So, the optimization metric is participation in OceanDAO?

Hi @MeMent0L, I don't want to say we have 1 metric. I don't believe that's constructive.

In orgs, there are usually a couple macro OKRs / KPIs everyone is working towards, while each individual has their own metric.
Currently, there are many metrics we observe and work towards improving at OceanDAO. 

Having said that, a key mission of OceanDAO is to nurture and grow the Ocean Ecosystem towards sustainability.
Therefore, I believe growing the total number of contributors is a very healthy, 101 metric that everyone can contribute towards.

I hope this helps, but I would love to hear your perspectives.

**Comment by MeMent0L** [comment]
I really like naive questions because they help get to the bottom of things :-) 

Here is my perspective, and then i promise i shut up! 

Absolutely agree with you on the importance of OceanDAO for the long term growth of the ecosystem. The question is how DAO participation KPIs stack vs. other KPIs like data consumption, data provision, LP/curation, new developments and partnerships, token appreciation. At this moment it's not readable to me. 

Explicitly ranking these different KPIs over time will provide the strategic focus that will make these design decisions like the burn, and other decisions, much, much easier. What comes first? OceanDAO participation? Data supply? Data usage? If it's immediate DAO participation, then a burn mechanism coupled with strong promotion/communication to stir participation could be the optimal design choice. If the first KPI is total # of projects building on Ocean, in the next 2 years, then it's best to not burn so that the DAO can fund more projects over time. 

Without that clarity (i.e. KPI ranking), design decisions typically end up as either (1) a compromise between each of the different entities' KPIs or (2) a call from upper management/authority because the decision was escalated. Neither of these processes is optimal. 

Strategy is often not about what to do, but what not to do. Explicitly ranking goals will make these tough choices trivial.

**Comment by idiom-bytes** [comment]
For the sake of continuing t move towards solutions, I just wanted to once again share the 2021 blog update. 

The blog post summarizes various intuitions about the designs/mechanism implemented.
I think it's important to go back to those intuitions.

Further, perhaps we need to improve overall DAO system & mechanism design, to document the reasons why the incentive engine looks the way it does. I think right now, the intuition behind why we're here (blog post) are important to be understood by everyone.

Please see my latest comments, in [#75] WRT To _Earmarks & Competition_

**Comment by trentmc** [comment]
Discord poll for this issue: https://discord.com/channels/612953348487905282/776848812534398986/915360169452535849

**Comment by MeMent0L** [comment]
For sure! I've said what i wanted to, and i participated in the poll. Thanks for your feedback.

**Comment by trentmc** [comment]
The Parameters & Roadmap WG met yesterday to finalize this issue. Discord votes are taken as an input sentiment, not as the final say. The final say is in the WG.

- Discord votes: 4 burn, 23 recycle
- Q: Any objections to “recycle”? (none)
- **Decision: recycle**
